.browserslistrc

```diff
@@ -0,0 +1,5 @@
+last 2 chrome versions
+last 2 firefox versions
+last 2 opera versions
+last 2 safari versions
+not dead
```

Comment:
We're missing several important ones here, namely Edge and the mobile (android/ios) versions of the ones listed.

I'd recommend using the defaults, then dropping IE which is 💀  and possibly other troublesome browsers.

```suggestion
defaults
not ie > 0
```